auth0/src/main/java/com/auth0/android/provider/IdTokenVerifier.java

```diff
@@ -0,0 +1,149 @@
+package com.auth0.android.provider;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.auth0.android.jwt.JWT;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import static android.text.TextUtils.isEmpty;
+
+class IdTokenVerifier {
```

Comment:
Class file was taken from mvc-commons implementation. Tests to be added on a separate PR